test/jdk/java/util/regex/RegExTest.java

```diff
@@ -963,58 +964,58 @@ public static void notCapturedGroupCurlyMatchTest() {
     public static void javaCharClassTest() {
         for (int i=0; i<1000; i++) {
             char c = (char)generator.nextInt();
-            check("{javaLowerCase}", c, Character.isLowerCase(c));
-            check("{javaUpperCase}", c, Character.isUpperCase(c));
-            check("{javaUpperCase}+", c, Character.isUpperCase(c));
-            check("{javaTitleCase}", c, Character.isTitleCase(c));
-            check("{javaDigit}", c, Character.isDigit(c));
-            check("{javaDefined}", c, Character.isDefined(c));
-            check("{javaLetter}", c, Character.isLetter(c));
-            check("{javaLetterOrDigit}", c, Character.isLetterOrDigit(c));
+            check("{javaLowerCase}", c, isLowerCase(c));
```

Comment:
There are more readable with the explicit Character class reference; with them one has to hunt around for the named method. It also makes it explicit what is being tested.

Reply:
I thought it was apparent since this is testing the *Character* properties and reduces the complexity, but if you think so maybe not. Reverting static import as I don't have a strong opinion on it.